backend/authentication/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
```

Comment:
I'm pretty sure this is mis-configured; this looks like the base `urls.py` file, but I think this should actually look different.

Reply:
We have updated both urls files, and both are working now.